stl/inc/xcharconv_ryu.h

```diff
@@ -391,7 +392,7 @@ _NODISCARD inline uint32_t __mulShift_mod1e9(const uint64_t __m, const uint64_t*
 #endif // ^^^ intrinsics unavailable ^^^
 }
 
-#define _WIDEN(_TYPE, _CHAR) (is_same_v<_TYPE,char> ? _CHAR : L##_CHAR)
+#define _WIDEN(_TYPE, _CHAR) _STD get<_TYPE>(_STD make_pair(_CHAR, L##_CHAR))

```

Comment:
I made this change after realizing the ternary operator was widening the result to `int`, even though the compiler seems to be fine with narrowing it back to the character type. But I've noticed that these calls to `get` and `make_pair` don't get inlined in debug mode. How much concern is there about debug performance vs. implicit widening/narrowing?

Reply:
The throughput hit from instantiating class and function templates here is icky. Why not go back to the original formulation and make the narrowing explicit by wrapping the value in `static_cast<_Type>(/**/)`?